DistanceRando-Spectrum/Metadata.cs

```diff
@@ -7,7 +7,7 @@ internal static class Metadata
 		public const string RandomizerVersion = "A.2.1";
 		public const string LogicVersion = "1.0-alpha2";
 
-		public const string MenuButtonText = "Distandomizer";
+		public const string MenuButtonText = "Distrandomizer";
```

Comment:
This works, although I think simplfying the button label would be cleaner.
```suggestion
		public const string MenuButtonText = "Randomizer";
```

Reply:
I wasn't sure about the text so feel free to change it :+1: